tests/main/snapctl-disable-services-install/task.yaml

```diff
@@ -0,0 +1,20 @@
+summary: Check that `snapctl stop --disable` actually stops services on install
```

Comment:
While at the top of the repo, can you run `./spread-shellcheck tests/main/snapctl-disable-services-install/task.yaml`?

Reply:
Done, fixed the issues with the for loop.